pkg/graveler/committed/merge_test.go

```diff
@@ -131,7 +131,7 @@ func Test_merge(t *testing.T) {
 				{rng: committed.Range{ID: "base:k3-k6", MinKey: committed.Key("k3"), MaxKey: committed.Key("k6"), Count: 2, EstimatedSize: 1234}},
 			}),
 			expectedResult: []testRunResult{{
-				mergeStrategies: []graveler.MergeStrategy{graveler.MergeStrategyNone, graveler.MergeStrategyDest, graveler.MergeStrategySource},
```

Comment:
Can we add a check to some of theses tests that the new metadata key was added correctly?

Reply:
I saw you added that in esti, so this comment becomes less critical.